### PR TITLE
source: the security slider is moved, not turned

### PR DESCRIPTION
**Pull request by dachary**
## Status

Ready for review

## Description of Changes

the security slider is moved, not turned

## Testing

N/A

## Deployment

N/A

### If you made changes to the server application code:

- [x] Linting (`make ci-lint`) and tests (`make -C securedrop test`) pass in the development container

### CODE REVIEW

File: securedrop/source_templates/index.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
     {% endassets %}
   </head>
   <body>
-    <div class="js-warning warning">{{ gettext('<strong>We recommend turning the Security Slider to Safest to protect your anonymity:</strong> <a id="disable-js" href="">Learn how to set it to safest</a>, or ignore this warning to continue.') }} <img id="js-warning-close" src="{{ url_for('static', filename='i/font-awesome/times-white.png') }}" width="12px" height="12px"></div>
+    <div class="js-warning warning">{{ gettext('<strong>We recommend moving the Security Slider to Safest to protect your anonymity:</strong> <a id="disable-js" href="">Learn how to set it to safest</a>, or ignore this warning to continue.') }} <img id="js-warning-close" src="{{ url_for('static', filename='i/font-awesome/times-white.png') }}" width="12px" height="12px"></div>
```
Comment (comradekingu) [inline securedrop/source_templates/index.html]: s/Learn how to set it to safest/Learn how to set it to "Safest"
Quotes could be used or not, there is another "Safest" here.

File: securedrop/source_templates/index.html [hunk 2]
```diff
@@ -89,11 +89,11 @@ <h2 class="welcome-text">
     Included here so the images can preload while the user is first
     reading the page. Hidden by default. #}
     <div class="bubble" id="security-slider-info-bubble">
-      <p>{{ gettext('You appear to be using the Tor Browser. You can turn the Security Slider to Safest in 4 easy steps!') }}</p>
+      <p>{{ gettext('You appear to be using the Tor Browser. You can move the Security Slider to Safest in 4 easy steps!') }}</p>
       <ol>
         <li>{{ gettext('Click the <img src="{icon}" alt="Tor icon"> Tor icon in the toolbar above').format(icon=url_for("static", filename="i/toronion.png"))  }}</li>
         <li>{{ gettext('Click <strong>Security Settings...</strong>') }}</li>
-        <li>{{ gettext('Turn the Slider to <strong>Safest</strong>, then click <strong>Ok</strong>') }}</li>
+        <li>{{ gettext('Move the Slider to <strong>Safest</strong>, then click <strong>Ok</strong>') }}</li>
```
Comment (comradekingu) [inline securedrop/source_templates/index.html]: Should be "OK"

File: securedrop/source_templates/index.html [hunk 3]
```diff
@@ -14,7 +14,7 @@
...
+    <div class="js-warning warning">{{ gettext('<strong>We recommend moving the Security Slider to Safest to protect your anonymity:</strong> <a id="disable-js" href="">Learn how to set it to safest</a>, or ignore this warning to continue.') }} <img id="js-warning-close" src="{{ url_for('static', filename='i/font-awesome/times-white.png') }}" width="12px" height="12px"></div>
     <div class="use-tor-browser warning">{{ gettext('<strong>We recommend using Tor Browser to access SecureDrop:</strong> <a id="recommend-tor" href="{tor_browser_url}">Learn how to install it</a>, or ignore this warning to continue.').format(tor_browser_url=url_for('info.recommend_tor_browser')) }} <img id="use-tor-browser-close" src="{{ url_for('static', filename='i/font-awesome/times-white.png') }}" width="12px" height="12px"></div>
```
Comment (comradekingu) [inline securedrop/source_templates/index.html]: Suggesting: "It is recommended" or "* recommends"

File: securedrop/source_templates/index.html [hunk 4]
```diff
@@ -89,11 +89,11 @@ <h2 class="welcome-text">
     Included here so the images can preload while the user is first
     reading the page. Hidden by default. #}
     <div class="bubble" id="security-slider-info-bubble">
-      <p>{{ gettext('You appear to be using the Tor Browser. You can turn the Security Slider to Safest in 4 easy steps!') }}</p>
+      <p>{{ gettext('You appear to be using the Tor Browser. You can move the Security Slider to Safest in 4 easy steps!') }}</p>
```
Comment (comradekingu) [inline securedrop/source_templates/index.html]: "You can adjust the security slider to "Safest", by doing these 4 things"
(avoids explaining it as "easy", which is redundant. Possibly "Adjust the security slider to "Safest" by" suffices)

File: securedrop/source_templates/index.html (hunk 2 above)
Comment (dachary) [reply securedrop/source_templates/index.html]: done!

File: securedrop/source_templates/index.html (hunk 1 above)
Comment (dachary) [reply securedrop/source_templates/index.html]: done

File: securedrop/source_templates/index.html (hunk 3 above)
Comment (dachary) [reply securedrop/source_templates/index.html]: done

**Pull request merged**